mmv1/products/cloudfunctions2/Function.yaml

```diff
@@ -423,6 +423,7 @@ properties:
               - !ruby/object:Api::Type::Integer
                 name: 'generation'
                 default_from_api: true
+                custom_flatten: 'templates/terraform/custom_flatten/cloudfunctions2_function_source_generation.go.erb'
```

Comment:
Can we add ignore_read: true to this field instead? From looking at the custom flatten function it seems to just read from state, which is pretty similar to just adding ignore_read